_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Maku brewing and Solita's passion pale ale
+author: Rinorragi
+excerpt: Solita ordered passion pale ale from Maku brewing and got introduction to their craftmenship
+tags: 
+- Beercraft
+---
+Solita wanted to have their own beer. We had our internal voting for the taste and name for the beer and "Intohimo" passion pale ale won the contest. At 21.6.2016 we got chance to taste 20l of the beer and make adjustments to it. All started with bus trip from Solita's office at Helsinki to Tuusula. We had great time already in the bus in discussion of projects and expectations of the upcoming beer. 
```

Comment:
Solita wanted to create a beer of their own (?)
we got a chance (?) tai We got an opportunity to taste the beer and give feedback until final brew (?)
with a bus trip
had a great time